src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -313,7 +320,7 @@ public void setDesiredPositions(double x, double y) {
       desiredYPosition.setDouble(y);
     }
     
-    setBaseReference(desiredBaseAngle);
+    setBaseReference(chooChooMap.toChooChooAngle(desiredBaseAngle));
```

Comment:
desiredBaseAngle is in radians, needs to be converted to degrees to lookup in the HashMap